Networking/WebSockets/WebSocketProtocol.hh

```diff
@@ -413,7 +413,7 @@ public:
                 // invalid reserved bits / invalid opcodes / invalid control frames / set compressed frame
                 if ((rsv1(frame) && !setCompressed(user)) || rsv23(frame) || (getOpCode(frame) > 2 && getOpCode(frame) < 8) ||
                     getOpCode(frame) > 10 || (getOpCode(frame) > 2 && (!isFin(frame) || payloadLength(frame) > 125))) {
-                    forceClose(user);
+                    forceClose(user, "cs-1");
```

Comment:
Can we provide the reason along with the some properties of the frame such as reserved bits, opcodes, control frames, compression, and payload length?

Reply:
Good point